vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -1091,4 +1091,8 @@ class VectorPreferences @Inject constructor(
                 .putBoolean(SETTINGS_THREAD_MESSAGES_SYNCED, shouldMigrate)
                 .apply()
     }
+
+    fun useLiveSenderInfo(): Boolean {
```

Comment:
Maybe rename to `showLiveSenderInfo`. So app side, we will have the user POV, so `show` and SDK side we will have the technical POV, so `use`.